tools/Resources.py

```diff
@@ -399,3 +364,65 @@ def get_latest_info_properties_multiple(target, token, uuids, propkey):
         else:
             print("Return code not 200 for " + str(propkey) + ": " + response.text)
             return False
+
+
+    def get_latest_stat_multiple(target, token, uuids, key):
+        if not isinstance(uuids, list):
+            print("Error in get multiple: uuids must be a list with multiple entries")
+            return False
+
+        # vrops can not handle more than 1000 uuids
+        uuids_chunked = list(chunk_list(uuids, 1000))
+        return_list = list()
+        url = "https://" + target + "/suite-api/api/resources/stats/latest/query"
+        headers = {
+            'Content-Type': "application/json",
+            'Accept': "application/json",
+            'Authorization': "vRealizeOpsToken " + token
+        }
+
+        m = MultipleThread0r()
+        thread_list = list()
+        for uuid_list in uuids_chunked:
+            t = Thread(target=m.get_chunk, args=(uuid_list, url, headers, key, target))
+            thread_list.append(t)
+            t.start()
+        for t in thread_list:
+            t.join()
+        return m.return_list
+
+# helper class to allow to thread
+class MultipleThread0r:
```

Comment:
I'd like us to be more serious about this.

Reply:
:_(